cluster_deploy/ext_tools/mysql/deploy_mysql_k8s.sh

```diff
@@ -0,0 +1,198 @@
+
+#! /bin/bash
+
+###
+# Copyright DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+
+CUR_SCRIPT_FOLDER=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+PULSAR_WORKSHOP_HOMEDIR=$( cd -- "${CUR_SCRIPT_FOLDER}/.." &> /dev/null && pwd )
+
+# TODO: Add usage stuff, parameter handling, etc.
+
+setupMysqlDebeziumDemo() { 
+
+
+# TODO: Make sure we have built project before we call this script since it requires Jars to be available, so make sure we're calling this function in the right place during deployment.
+
+# Expecting first parameter to be the scenario name:
+scnName=$1
+
+
+source "${PULSAR_WORKSHOP_HOMEDIR}/_bash_utils_/utilities.sh"
+
+scnHomeDir="${PULSAR_WORKSHOP_HOMEDIR}/scenarios/${scnName}"
```

Comment:
Anything under `cluster_deploy` should be scenario agnostic. In my opinion, the script here should be only used for deploying MySQL in K8s. I don't recommend we put scenario specific stuff here

Having said that, we probably need to extend `deployScenario.sh` to add support for deploying source/sink connectors.